docs/UserGuide.md

```diff
@@ -476,3 +533,63 @@ Parameter | Prefix | Constraints, Examples
 **TAG** | `t/` | Tags names should be alphanumeric and have a maximum of 45 characters. <br> e.g. `t/important`
 **SUGGESTION_TYPE** | `by/` | Suggestion types can only be either `available`, `contract` or `frequency`. <br> e.g. `by/available`
 **INDEX** | - | Index is a number greater than 0 that is based on the numberings beside each client or note. <br> e.g. `1` would refer to the first client or note.
+
+--------------------------------------------------------------------------------------------------------------------
+## Appendix A - Introduction to the UI
+
+### Start Page
+
+The diagram below shows the key UI elements of TBM upon first start-up.
+
+![Annotated UI Default Page](images/command-screenshots/UI_annotated.png)
+
+* **Command Box** allows you to input  commands in their correct syntax to interact with TBM.
+
+* **Client List Panel** will always allow you to view your client's information at a go. It's scrollable so that you 
+never have to worry about not being able to see all your important clients at a single glance!
+
+* **Client Card** will show you brief information a particular client. 
+
+* **Result Display** will show you the output of executing your command.
+    * It indicates successful commands.
+    
+    * On the off-chance there's a syntax issue, TBM will remind you of the correct syntax it expects, something like this:
+ 
+     ![Result Display slowing error message](images/command-screenshots/result_display_error_syntax.png)
+     
+* **Generic Widget View Box** is aptly named because it shall spawn *Client Cards* or *Country Cards* when
+ relevant commands are sent. Initially it will show you your local time, to the precise second and your own location.
+   
+     
+
+
+### UI when Viewing Clients
+
+The diagram below is what you should see upon viewing a particular client (i.e. firing up the command `client view 1
+`). It shows how to identify a particular  client's  `CLIENT_INDEX`, and an  associated note's `CLIENT_NOTE_INDEX`.
+
+![Annotated Client View UI Layout](images/command-screenshots/client_view_annotated.png)
+
+Here, the **Client Card** displaying the note that's tagged as `hobbies` is referred to by the `CLIENT_INDEX` of `1`.
+
+Being able to identify these is key to adding, deleting and editing client notes as well as editing clients.
```

Comment:
This part should be made clearer with use of actual names in the diagram. The "Client Card" as mentioned above refers to the display on the right but it's mentioned here to be "displaying the note".

Reply:
Will add the labels of `Country Note Card` and `Client Note Card` and make that reference